This error
```
2026-05-02T17:40:07.3262690Z /opt/local/var/macports/build/shichizip-367fcc89/work/shichizip-v0.1.5-source/ShichiZip/QuickActions/Extensions/QuickActionRequestHandlers.swift:23:9: error: passing closure as a 'sending' parameter risks causing data races between code in the current task and concurrent execution of the closure
2026-05-02T17:40:07.3263610Z         Task {
2026-05-02T17:40:07.3263750Z         ^~~~~~
```
was fixed in the latest Xcode. Can someone please update macOS 26 CI to Xcode 26.4?

--- a/archivers/shichizip/Portfile
+++ b/archivers/shichizip/Portfile
@@ -1,0 +1,58 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               xcode 1.0
+
+github.setup            idawnlight ShichiZip 0.1.5 v
+name                    [string tolower ${github.project}]
+github.tarball_from     releases
+distname                ${name}-${github.tag_prefix}${version}-source
+revision                0
+platforms               {darwin >= 23}
+categories              archivers aqua
+maintainers             {i0ntempest @i0ntempest} openmaintainer
+license                 LGPL-2.1
+platforms               macosx
+distname                ${name}-${github.tag_prefix}${version}-source
+
+description             7-Zip FM derivative intended for macOS
+long_description        ${github.project} is the missing 7-Zip derivative intended for macOS aiming\
+                        to provide the 7-Zip File Manager experience, rebuilt natively for macOS.
+
+checksums               rmd160  95f26b2d30575483661aa65f0a5982e61c386f81 \
+                        sha256  e1aea7663c18e753c80079af3f7c245d663fd3f3adcdf947b16b72e7c092e4b9 \
+                        size    10255031
+
+# switch to zig once emutls bug is fixed
+depends_build-append    port:zig-bootstrap
+
+extract.rename          yes
+
+# bundled sfx stub is always Windows x86
+if {![variant_isset zstd]} {
+    # 7zip standard
+    set variant_suffix  ""
+    set zig_variant_arg ""
+} else {
+    # 7zip zstandard variant
+    set variant_suffix  zs
+    set zig_variant_arg -Dvariant=${variant_suffix}
+}
+
+pre-build {
+    system -W ${worksrcpath} "${prefix}/bin/zig-bootstrap build -j${build.jobs} lib ${zig_variant_arg} -Doptimize=ReleaseFast --verbose --verbose-link --verbose-cc --summary all -p build"
+    system -W ${worksrcpath} "${prefix}/bin/zig-bootstrap build -j${build.jobs} sfx ${zig_variant_arg} -Doptimize=ReleaseSmall --verbose --verbose-link --verbose-cc --summary all -p build"
+}
+
+universal_variant       no
+xcode.scheme            "${github.project}[string toupper ${variant_suffix}]"
+xcode.configuration     Release
+xcode.build.settings-append \
+                        CODE_SIGN_IDENTITY=-
+
+destroot {
+    file copy {*}[glob ${worksrcpath}/build/${xcode.configuration}/${github.project}*.app] ${destroot}${applications_dir}
+}
+
+variant zstd description {use 7zip Zstandard variant, add support for Zstd/Brotli/Lizard/Fast LZMA2 codecs} {}

--- a/lang/zig-bootstrap/Portfile
+++ b/lang/zig-bootstrap/Portfile
@@ -1,0 +1,88 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               codeberg 1.0
+
+set base_name           zig
+codeberg.setup          ziglang ${base_name}-bootstrap 0.16.0
+revision                0
+categories              lang
+license                 MIT
+maintainers             {i0ntempest @i0ntempest} openmaintainer
+homepage                https://ziglang.org/
+
+# https://ziglang.org/download/0.16.0/release-notes.html#OS-Version-Requirements
+platforms               {darwin >= 22}
+
+description             Zig programming language (bootstrap build with no runtime dependencies)
+long_description        Zig is a general-purpose programming language designed for robustness, \
+                        optimality, and maintainability. This is the bootstrap build version with \
+                        no external runtime dependencies.
+
+checksums               rmd160  474f5ba6be808a5c6413727e9682cd25dffeeb2f \
+                        sha256  a3d5867c25cb2046568b12e373219c35e49d02a7fabd21a4e8acc527fc4521c4 \
+                        size    84043297
+
+# sterilize MacPorts build environment
+compiler.cpath
+compiler.library_path
+
+configure.cxx_stdlib
+
+configure.cflags
+configure.cxxflags
+configure.cppflags
+configure.optflags
+configure.ldflags
+
+configure.universal_cflags
+configure.universal_cxxflags
+configure.universal_cppflags
+configure.universal_ldflags
+configure.universal_args
+
+configure.ccache        no
+configure.distcc        no
+
+# only supply cmake; use make, python3, and clang(++) from the system/CLT.
+compiler.whitelist      clang
+depends_build-append    port:cmake
+
+if {${build_arch} eq "x86_64"} {
+    set arch_filename   ${build_arch}
+} else {
+    # file name is "aarch64" for arm64
+    set arch_filename   aarch64
+}
+
+extract.rename          yes
+
+patchfiles-append       patch-parallel.diff
+post-patch {
+    reinplace -E "s|^(\[\[:space:\]\]*)cmake(\[\[:space:\]\]+)|\\1${prefix}/bin/cmake\\2|" ${worksrcpath}/build
+    # macOS github runner has only 8GB of memory
+    reinplace "s|max_rss = 8_000_000_000|max_rss = 7_500_000_000|g" ${worksrcpath}/zig/build.zig
+}
+
+set abi                 none
+set mcpu                baseline
+use_configure           no
+build.env               PATH=/usr/bin:/bin:/usr/sbin:/sbin \
+                        CC=${configure.cc} \
+                        CXX=${configure.cxx}
+build.cmd               ./build
+build.target            ${arch_filename}-macos-${abi}
+build.args              ${mcpu} ${build.jobs}
+
+destroot {
+    xinstall -d ${destroot}${prefix}/libexec/${name}
+    xinstall -m 0755 ${worksrcpath}/out/${base_name}-${arch_filename}-macos-${abi}-${mcpu}/${base_name} ${destroot}${prefix}/libexec/${name}/
+    copy ${worksrcpath}/out/${base_name}-${arch_filename}-macos-${abi}-${mcpu}/lib ${destroot}${prefix}/libexec/${name}/
+    set wrapper ${destroot}${prefix}/bin/${name}
+    set fh [open ${wrapper} w]
+    puts ${fh} "#!/bin/sh"
+    puts ${fh} "export ZIG_LIB_DIR=${prefix}/libexec/${name}/lib"
+    puts ${fh} "exec ${prefix}/libexec/${name}/${base_name} \"\$@\""
+    close ${fh}
+    file attributes ${wrapper} -permissions 0755
+}

--- a/lang/zig-bootstrap/files/patch-parallel.diff
+++ b/lang/zig-bootstrap/files/patch-parallel.diff
@@ -1,0 +1,64 @@
+--- build	2026-04-14 04:52:27
++++ build	2026-05-03 21:06:47
+@@ -5,7 +5,12 @@
+ 
+ TARGET="$1" # Example: riscv64-linux-gnu
+ MCPU="$2" # Examples: `baseline`, `native`, `generic+v7a`, or `arm1176jzf_s`
++JOBS="${3:-}" # Optional: number of parallel jobs
+ 
++if [ -z "$JOBS" ]; then
++  JOBS="$(getconf _NPROCESSORS_ONLN 2>/dev/null || nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || echo 1)"
++fi
++
+ ROOTDIR="$(pwd)"
+ ZIG_VERSION="0.16.0"
+ 
+@@ -56,7 +61,7 @@
+   -DCLANG_TOOL_CLANG_LINKER_WRAPPER_BUILD=OFF \
+   -DCLANG_TOOL_C_INDEX_TEST_BUILD=OFF \
+   -DCLANG_TOOL_LIBCLANG_BUILD=OFF
+-cmake --build . --target install
++cmake --build . --target install --parallel "$JOBS"
+ 
+ # Now we build Zig, still with system C/C++ compiler, linking against LLVM,
+ # Clang, LLD we just built from source.
+@@ -67,7 +72,7 @@
+   -DCMAKE_PREFIX_PATH="$ROOTDIR/out/host" \
+   -DCMAKE_BUILD_TYPE=Release \
+   -DZIG_VERSION="$ZIG_VERSION"
+-cmake --build . --target install
++cmake --build . --target install --parallel "$JOBS"
+ 
+ # Now we have Zig as a cross compiler.
+ ZIG="$ROOTDIR/out/host/bin/zig"
+@@ -89,7 +94,7 @@
+   -DCMAKE_RC_COMPILER="$ROOTDIR/out/host/bin/llvm-rc" \
+   -DCMAKE_AR="$ROOTDIR/out/host/bin/llvm-ar" \
+   -DCMAKE_RANLIB="$ROOTDIR/out/host/bin/llvm-ranlib"
+-cmake --build . --target install
++cmake --build . --target install --parallel "$JOBS"
+ 
+ # Same deal for zstd.
+ # The build system for zstd is whack so I just put all the files here.
+@@ -97,6 +102,7 @@
+ cp "$ROOTDIR/zstd/lib/zstd.h" "$ROOTDIR/out/$TARGET-$MCPU/include/zstd.h"
+ cd "$ROOTDIR/out/$TARGET-$MCPU/lib"
+ $ZIG build-lib \
++  -j"$JOBS" \
+   --name zstd \
+   -target $TARGET \
+   -mcpu=$MCPU \
+@@ -190,11 +196,12 @@
+   -DCLANG_TOOL_C_INDEX_TEST_BUILD=OFF \
+   -DCLANG_TOOL_LIBCLANG_BUILD=OFF \
+   -DLLD_BUILD_TOOLS=OFF
+-cmake --build . --target install
++cmake --build . --target install --parallel "$JOBS"
+ 
+ # Finally, we can cross compile Zig itself, with Zig.
+ cd "$ROOTDIR/zig"
+ $ZIG build \
++  -j"$JOBS" \
+   --prefix "$ROOTDIR/out/zig-$TARGET-$MCPU" \
+   --search-prefix "$ROOTDIR/out/$TARGET-$MCPU" \
+   -Dflat \

